docs/installation.md

```diff
@@ -163,6 +163,20 @@ chmod +x <cmsis-toolbox-installation-dir>/bin/cbuild
 ...
 ```
 
+### Registering CMSIS_PACK_ROOT with cpackget
+
+Once you are done with setting up the environment variables, run the following on the command line:
+
+```Shell
+cpackget init --pack-root path/to/mypackroot https://www.keil.com/pack/index.pidx
```

Comment:
Please note that you don't have to specify the --pack-root unless you want to override the CMSIS_PACK_ROOT environment variable from the command line.